doc/ref_arch/README.md

```diff
@@ -47,6 +47,22 @@ RA1 is focussing on an OpenStack Reference Architecture that will support VM-bas
 
 The scope of RA2 will be to enable support for containerised and VM-based workloads (i.e. VMs managed by Kubernetes). A thorough gap analysis will need performing to understand the level to which Kubernetes might support traditional VNFs (i.e. to what level can Kubernetes conform to the VIM specification from ETSI NFV v3, or the Vi-Vnfm interface, for example).
 
-RA2 is delivering a Kubernetes Reference Architecture that will be standalone, meaning:
-- If there are any components from RA1 (or any other subsequent RA) that are needed in RA2, they will be explicitly included rather than requiring a Reference Implementation to refer to multiple Reference Architectures
-- The Kubernetes Reference Architecture will also include the Kubernetes cloud providers on which a Reference Implementation can choose to deploy an RA2-conformant Kubernetes
+RA2 is delivering a Kubernetes Reference Architecture that will be standalone, meaning that a Reference Implementation for RA2 can be established solely on the content and requirements present in the RA2 documentation.
+
+Kubernetes can be executed on top of different types of underlying infrastrucutre. Examples of those underlying infrastructures are:
+- Infrastructure as a Service based on virtual machines such as OpenStack in general or a Reference Implementation of RA1
+- Physical servers. (This is often referred to as "baremetal Kubernetes")
+
+Kubernetes expects certain capabilities or resources provided by the underlying infrastructure. Examples for resources or capabilities of the underlying infrastructure consumed by Kubernetes are, but are not limited to:
+- storage (ephemeral/persistent)
+- connectivity
+- compute
+
+RA2 contains concrete provisions on the type of infrastructure resources consumed by Kubernetes.
+
+If the underlying infrastructure provides a dynamic way to allocate resources, Kubernetes can integrate directly with the underlying infrastructure and use it as a infrastructure resource provider. Kubernetes uses a plugin-based mechanism for integration with any type of infrastructure resource providers. Most prominently Kubernetes Cloud Provider collect various integrations towards a particular infrastructure platform e.g. to create volumes or additional worker nodes. The Kubernetes cluster api harmonizes some of the k8s cluster management related integrations with different infrastructure platforms under a common API. With the Container Network Interface (CNI) and the Container Storage Interface (CSI) specifications Kubernetes also provides additional means to integrate with the underlying infrastructure. (Note: the use of CNIs or CSIs does not imply the use of specific platform resources.)
+
```

Comment:
What seems to be assumed here is that the OS is inherently part of the RA2 box (since the box beneath is virtual or real "hardware"). I think we should be explicit in that inclusion (at least point it out in the text). The reason why I bring this up is because I believe we need to make it explicit that the choice of OS (in RIx of RA2) needs to be independent from the choice of OS used to generate the docker images.

Reply:
Yes, the OS (for the Kubernetes distribution) should be part of an RI of RA2. The relation to the OS used to create the docker images for the workloads (I assume that's what you mean) is not part of the relation to the underlying infra. Thus I wonder if this should rather be described in RA2 rather than in the readme for both RAs.
If there is a need to have a more detailed picture of RA2, underlying infra and the workload for RA2 I'm willing to create a PR for that as well.

Reply:
Ok, let me check how/where this would be logical to be included in RA2. Maybe it's there already....